### ISSUE TITLE
10 HTML Elements You Didn't Know You Needed 

### ISSUE THREAD

**Issue by nirjan-dev** [nirjan-dev/learning-to-code#211]
[link to the post](https://dev.to/emmawedekind/10-html-element-you-didnt-know-you-needed-3jo4)

# Audio 

The <audio> tag defines a sound, such as music or other audio streams. There are three currently supported file formats: MP3, WAV, and OGG.

# Blockquote 
The <blockquote> tag specifies a section that is quoted from another source.

# Output
The <output> tag represents the result of a calculation. You can use the plus sign and equal symbol to indicate that the first and second input values should be “outputted” to the output tag; you can denote this with a for attribute containing the ids of the two elements to combine.

```js


<form oninput="totalWeight.value=parseInt(catAWeight.value)+parseInt(catBWeight.value)">0
  <input type="range" id="catAWeight" value="50">100
  +<input type="number" id="catBWeight" value="50">
  =<output name="totalWeight" for="catAWeight catBWeight"></output>
</form>

```

# Picture 

The <picture> tag allows you to specify image sources. Instead of having an image which you scale up and down depending upon the viewport width, multiple images can be designed to fill the browser viewport.

The picture tag contains two different tags: one or more <source> elements and one <image> element.

The source element has the following attributes:

    srcset (required): Defines the URL of the image to display
    media: Accepts any valid media query that you might define within CSS
    sizes: Defines a single width value, a single media query with width value, or a comma separated list of media queries with a width value
    type: Defines the MIME type.

The browser uses the attribute values to load the most appropriate image; it uses the first <source> element with a matching hit and ignores the subsequent source elements.

The <img> tag is used to provide backwards compatibility if a browser doesn’t support the element or if none of the <source> tags match.

![image](https://user-images.githubusercontent.com/29270715/73426014-698aa400-435b-11ea-86a5-82a049de1d48.png)

# Progress 
The <progress> tag represents the progress of a task.

The <progress> tag is not a replacement for the <meter> tag, thus components indicating disk space usage or query result relevance should use the <meter> tag.
![image](https://user-images.githubusercontent.com/29270715/73426121-a22a7d80-435b-11ea-82e3-173c744f99aa.png)

# Meter 
The <meter> tag defines a scalar measurement within a defined range, or a fractional value. You can also refer to this tag by the name “gauge.”

You can use the <meter> tag to display disk usage statistics or to indicate the relevance of search results.

The <meter> tag should not be used to indicate progress of a task; these types of components should be defined by a <progress> element.

![image](https://user-images.githubusercontent.com/29270715/73426111-9ccd3300-435b-11ea-94b5-36f0108f1d2a.png)

# Template 
The <template> tag contains content that is hidden from the user, but will be used to instantiate HTML code repeatedly.

Using JavaScript, you can render this content with the cloneNode() method.

template

![image](https://user-images.githubusercontent.com/29270715/73426172-bd958880-435b-11ea-8091-8a32dd9ba367.png)

# Time 
The <time> tag defines a human-readable date or time. This can be used to encode date and times in a machine-readable manner so that user agents can add birthday reminders or scheduled events to the user’s calendar. Additionally, this allows search engines to produce “smarter” search results.

```js


<p>My cat wakes up at <time>11:00</time> each day.</p>

```

![image](https://user-images.githubusercontent.com/29270715/73426231-e453bf00-435b-11ea-9d11-1a6e20bf10ec.png)

# Video 
The <video> tag specifies a movie clip or video stream. The supported formats include MP4, WebM, and Ogg.

![image](https://user-images.githubusercontent.com/29270715/73426262-f6356200-435b-11ea-834b-4aad127e9653.png)

# Word Break Opportunity 

If you have a long block of text, or a long word, you can use the <wbr> tag to specify where in a body of text it would be ideal to break. This is one way to ensure the browser, upon resize, doesn’t break your text in a strange spot.

![image](https://user-images.githubusercontent.com/29270715/73426303-1107d680-435c-11ea-9eb7-20f48fe49a97.png)
